src/Features/Core/Portable/NavigateTo/NavigateToMatchKind.cs

```diff
@@ -9,6 +9,11 @@ internal enum NavigateToMatchKind
         Prefix = 1,
         Substring = 2,
         Regular = 3,
-        None = 4,
+        CamelCaseExact = 4,
+        CamelCasePrefix = 5,
+        CamelCaseNonContiguousPrefix = 6,
+        CamelCaseSubstring = 7,
+        CamelCaseNonContiguousSubstring = 8,
+        Fuzzy = 9
```

Comment:
FYI @minestarks and @brettfo  Note: this doesn't break you at all.  However, it enables you to provide much more fine grained information about your NavigateTo matches if you support fuzzy matching or camel case matching.

Reply:
Let me know if you need any more info.

Reply:
Do we have any concern that None is being replaced with CamelCaseExact ?

Reply:
I was thinking about it too. Previously None was silently converted to Regular in by GetNavigateToMatchKind(), now it's going to be become CamelCaseExact. It's not that anybody probably ever create a result with None match kind, but still I think it would be safer to keep it and still convert to Regular (Fuzzy now).

Reply:
Sounds good.  I fixed it in both places for next review.